src/Avalonia.Input/PointerEventDetails.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Avalonia.Input
+{
+    public class PointerEventDetails
```

Comment:
Created this only to group pointer related arguments passed to Clear/SetPointerOver. So we can avoid hell of arguments, which potentially can only grow.